app/code/Magento/Sales/view/adminhtml/web/order/create/scripts.js

```diff
@@ -1170,8 +1170,10 @@ define([
 
         submit : function()
         {
-            jQuery('#edit_form').trigger('processStart');
-            jQuery('#edit_form').trigger('submitOrder');
+            if(jQuery('#edit_form').valid()) {
```

Comment:
Hi @passtet, please add space between `if` and `(`. 
And what about move `jQuery('#edit_form')` to a variable? Then use it to validate and trigger methods.

Reply:
@williankeller done. I would actually refactore the whole file for those, but I guess that there would be too many conflicts :)